tests/mongo/test_mongo_translator.py

```diff
@@ -27,30 +123,43 @@ def test_mongo_expression():
         ]
     }
 
-    result = MongoExpression().parse(query)
+    result = permission_conditions_to_mongo_query(query)
     assert result == expected
 
 
-def test_mongo_expression_with_jinja():
-    expr = "(type == 'YTD') or (periode == 'yo_{{periode}}')"
-    expected = {'$or': [{'type': 'YTD'}, {'periode': 'yo_{{periode}}'}]}
-    assert MongoExpression().parse(expr) == expected
-
-    expr = "(type == 'YTD') or (periode == {{periode}})"
-    expected = {'$or': [{'type': 'YTD'}, {'periode': '{{periode}}'}]}
-    assert MongoExpression().parse(expr) == expected
-
-    expr = "(type == 'YTD') or (periode == 'yo_{{my_indic[\"a\"]}}')"
-    expected = {'$or': [{'type': 'YTD'}, {'periode': 'yo_{{my_indic["a"]}}'}]}
-    assert MongoExpression().parse(expr) == expected
+def test_permission_conditions_to_mongo_query_with_jinja():
```

Comment:
I have trouble understanding the point of these tests, as we are not interpolating the variables in curly braces here.
@PrettyWood do you know why it was necessary to test that in the first place ? Is that to verify that the values are indeed not interpolated ?